### PR TITLE
testing/py3-us: modernize

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

**Comment by maxice8** [comment]
@fabaff

File: testing/py3-us/APKBUILD [hunk 1]
```diff
@@ -11,18 +11,15 @@ license="BSD-3-Clause"
 replaces="py-us" # for backwards compatibility
 provides="py-us=$pkgver-r$pkgrel" # for backwards compatibility
 depends="py3-jellyfish py3-setuptools python3"
```
Comment (TBK) [inline testing/py3-us/APKBUILD]: Looks incorrect to me. The makedepends should remain, the overlap should be removed from depends.

Would also be great if check was added - https://github.com/unitedstates/python-us/tree/master/tests

Comment (maxice8) [reply testing/py3-us/APKBUILD]: No, assuming dependency distribution was correct before then you should ever remove from makedepends when against depends.

Comment (TBK) [reply testing/py3-us/APKBUILD]: Forgot to paste link for reqs - https://github.com/unitedstates/python-us/blob/master/requirements.txt

**Comment by maxice8** [comment]
On Tue, May 28, 2019 at 01:42:27PM -0700, TBK wrote:
> TBK commented on this pull request.
> 
> 
> 
> > @@ -11,18 +11,15 @@ license="BSD-3-Clause"
>  replaces="py-us" # for backwards compatibility
>  provides="py-us=$pkgver-r$pkgrel" # for backwards compatibility
>  depends="py3-jellyfish py3-setuptools python3"
> 
> Forgot to paste link for reqs - https://github.com/unitedstates/python-us/blob/master/requirements.txt
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/alpinelinux/aports/pull/8100#discussion_r288293435

py3-setuptools is required because /usr/bin/states provided by the package imports pkg_resources and python3 is required because there is a script (/usr/bin/states) that has a shebang on python3

**Pull request closed without merging**